website/addons/evernote/static/evernoteLogActionList.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please also add an `evernoteAnonymousLogActionList.json` that contains anonymized language for these logs, e.g. `"A user linked an Evernote NoteBook to a node"`. This hasn't gone in yet, but it will soon. See https://github.com/CenterForOpenScience/osf.io/pull/5825/ for reference